integrations/Refiner/browser.js

```diff
@@ -78,8 +78,8 @@ class Refiner {
     logger.debug("===In Refiner Group===");
     const { message } = rudderElement;
     const { userId, groupId, traits } = message;
-    const email = message.traits?.email || message.context?.traits?.email;
-    if (!userId && !email) {
+    const userEmail = message.context?.traits?.email;
```

Comment:
Are skipping `traits` ?

Reply:
@utsabc yes, because this email is user level email, in group call if someone want to send an email for an account level then they will send in traits. basically what is happening is if we don't separate this then user level email value is replaced by an account level email. so to keep separate if we found email value in context.traits then we will send it at user level or else the email which is inside traits will be sent at account level.

Reply:
Got it